kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -184,6 +184,11 @@ private String getRowId(SinkRecord record) {
 
   @Override
   public void put(Collection<SinkRecord> records) {
+    // check for non-retriable errors and fail the task if any.
+    // adding this because any Exception thrown in flush will be ignored by the framework, which
```

Comment:
make the comment more relevant to when it'll eventually be committed by explaining why we want to throw an exception in put rather than why it's not sufficient to throw an exception in flush.